src/support/utf8.c

```diff
@@ -232,7 +232,14 @@ size_t u8_offset(const char *s, size_t charnum)
     return i;
 }
 
-/* byte offset => charnum */
+/**
+ * @brief      Calculates number of characters in a UTF-8 string
+ * 
+ * @param[in]  iStr UTF-8 encoded string
```

Comment:
The parameter is called `s`, not `iStr`. The convention in this file seems to be to use short lower case identifiers anyway.